docs/cel2/l2-operator-guide.md

```diff
@@ -24,61 +24,30 @@ Therefore, operators looking to run full archive nodes or serve requests for his
 Since the Celo L2 node does still require the full pre-migration chain data, these operators will require approximately double the storage space as is currently needed.
 
 :::warning
-The instructions below are not yet fully usable, as some values need to be set. They are marked as TODO.
+The instructions below are not yet fully useable, as some values need to be set.
 :::
 
-## Stopping an L1 Node
+### Starting an L2 node
```

Comment:
1. I would keep the stopping instructions as a first test of them.
2. I made the headings level 2 instead of 3 to get a more usable TOC (didn't want to mess with the general settings of them).

Reply:
Agree. I deleted changes with a merge conflict